### ISSUE TITLE
ACE fails if :root {   color-scheme: light dark; } is used

### ISSUE THREAD

**Issue by Lululalu** [daisy/ace#468]
:root {
  color-scheme: light dark;
}

It makes Ace confused about color contrast. It compares the default colors against a white background, not a black one.
